src/helpers/issue.ts

```diff
@@ -118,7 +118,8 @@ export const getAllIssueComments = async (issue_number: number): Promise<Comment
 
       await checkRateLimitGit(response?.headers);
 
-      if (response.data) {
+      // Fixing infinite loop here, it keeps looping even when its an empty array
+      if (response.data && response.data.length > 0) {
```

Comment:
`It keeps looping even when its an empty array` --- I am not sure whether it keep looping if its an empty array but also interesting. If you have more bandwidth, would you be posting the simple proof like 1 ~ 2 line of javascript code

Reply:
OKay, I'd go with the concise suggestion from pavlovcik, but I'd do that next time.

I ran the payout command and saw it loop endlessly so just a quick fix